CONTRIBUTING.md

```diff
@@ -48,11 +44,9 @@ Once a pull request or issue have been submitted, anyone can comment or vote on
 - It's okay to have input without voting
 - Silence gives assent
 
-A pull request with at least two **+1** votes, no **-1** votes, and that has been open for at least 3 days, is ready to be merged. The merge should be done by someone from a different organization than the submitter. (We sometimes waive the 3 days for cosmetic-only changes -- use good judgment.)
-
-If an issue gets any **-1** votes, the comments on the issue need to reach consensus before the issue can be resolved one way or the other. There isn't any strict time limit on a contentious issue.
+A pull request with at least two **+1** votes, no **-1** votes, that has been open for at least 3 days, and whose cross-referenced pull requests to server and compliance have similarly been upvoted is ready to be merged. The merge should be done by someone from a different organization than the submitter. (We sometimes waive the 3 days for cosmetic-only changes -- use good judgment.) A pull request to either the schemas, servers or compliance repository that involves changes to the others should not be merged without coordinating, mergable pull requests to the other repositories. Similarly, if an issue gets any **-1** votes, the comments on the issue need to reach consensus before the issue can be resolved one way or the other. There isn't any strict time limit on a contentious issue.
```

Comment:
As noted above, could we just solve this by requiring that the same individual handle all three merges at the same time - this would make the responsibility for making sure the merges are consistent be easier to manage.